packages/loot-core/src/client/actions/modals.ts

```diff
@@ -36,8 +37,7 @@ export function replaceModal<M extends ModalType>(
   name: M,
   options?: FinanceModals[M],
 ): ReplaceModalAction {
-  // @ts-expect-error TS is unable to determine that `name` and `options` match
-  const modal: M = { name, options };
+  const modal: Modal = { name, options };
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏Amazing